### DISCUSSION TITLE
Cannot inference with deployed TensorRT engine on Jetson AGX Xavier.

### DISCUSSION THREAD

**Discussion by tung18tht** [open-mmlab/mmdeploy#1513] [Q&A] [answered]
Hi, I am trying to setup the mmdeploy on my Jetson AGX Xavier. Everything went fine including the conversion step.
I converted the yolox-s detection model and got the `end2end.engine` as the output.

But when I try to inference with the command `./build/bin/object_detection cuda ./end2end.engine ./demo.jpg`, it has the following error:
```
[2022-12-10 14:01:29.475] [mmdeploy] [error] [model.cpp:39] Failed to load model: "./end2end.engine", implementations tried: [("DirectoryModel", 0)]
[2022-12-10 14:01:29.475] [mmdeploy] [error] [model.cpp:16] Failed to load model "./end2end.engine"
[2022-12-10 14:01:29.475] [mmdeploy] [error] [model.cpp:21] failed to create model: not supported (2) @ .../mmdeploy/csrc/mmdeploy/core/model.cpp:41
failed to create detector, code: 6
```

I already tried reinstalling, and using python API, but the problem persists. Do you guys have any idea on the problem and how to fix it?

**Comment by lvhan028** [comment, marked as answer]
Please read https://github.com/open-mmlab/mmdeploy/blob/master/docs/en/get_started.md

`build/bin/object_detection` is implemented on MMDeploy Inference SDK.
SDK performs model inference with not only the backend engine file but also  **meta info**.

How to get meta info? 
Use `--dump-info` when converting an OpenMMLab model by `tools/deploy.py`.
The generated meta info, as well as the backend engine file, are located in the directory specified by `--work-dir`
They make up what we call **MMDeploy Model**.

After exporting **MMDeploy Model**, pass the path specified by `--work-dir` to `build/bin/object_detection`
